pkg/utils/k8sutil_test.go

```diff
@@ -1,246 +1,289 @@
 package utils
 
 import (
-	"os"
-	"regexp"
+	"reflect"
 	"strings"
 	"testing"
 
 	"github.com/kubeless/kubeless/pkg/spec"
 	k8sErrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
 	"k8s.io/client-go/kubernetes/fake"
 	"k8s.io/client-go/pkg/api"
 	"k8s.io/client-go/pkg/api/v1"
 	av2alpha1 "k8s.io/client-go/pkg/apis/autoscaling/v2alpha1"
-	"k8s.io/client-go/pkg/apis/batch/v2alpha1"
 	xv1beta1 "k8s.io/client-go/pkg/apis/extensions/v1beta1"
 	"k8s.io/client-go/rest"
 	ktesting "k8s.io/client-go/testing"
 )
 
-func findAction(fake *fake.Clientset, verb, resource string) ktesting.Action {
-	for _, a := range fake.Actions() {
-		if a.Matches(verb, resource) {
-			return a
+func getEnvValueFromList(envName string, l []v1.EnvVar) string {
+	var res v1.EnvVar
+	for _, env := range l {
+		if env.Name == envName {
+			res = env
+			break
 		}
 	}
-	return nil
+	return res.Value
 }
 
-func hasAction(fake *fake.Clientset, verb, resource string) bool {
-	return findAction(fake, verb, resource) != nil
+func copyFunc(orig, dest *spec.Function) {
+	*dest = *orig
```

Comment:
Would it be clearer (particularly with respect to argument order) if the the callers just did that assignment explicitly instead of using this function? (I think so)